src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -370,4 +376,11 @@ trait ResourceRoutes extends UserInfoDirectives with SecurityDirectives with Sam
         complete(resourceService.listResourceChildren(resource, samRequestContext).map(children => StatusCodes.OK -> children))
       }
     }
+
+  def deletePolicy(policyId: FullyQualifiedPolicyId, userInfo: UserInfo, samRequestContext: SamRequestContext): server.Route =
+    delete {
+      requireOneOfAction(policyId.resource, Set(SamResourceActions.alterPolicies, SamResourceActions.deletePolicy(policyId.accessPolicyName)), userInfo.userId, samRequestContext) {
+        complete(resourceService.deletePolicy(policyId, samRequestContext).map(_ => StatusCodes.NoContent))
```

Comment:
Just to be clear, for now, we are not checking if this policy applies to any children, even though we know that we should check that prior to deleting.  I'm OK with this if that's what was decided for now.  Do we have a ticket to come back and implement this later?  If so, I recommend putting a comment with a TODO here pointing to that ticket.  Either here or in `ResourceService.deletePolicy`.